content/articles/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/index.md

```diff
@@ -0,0 +1,79 @@
+Setting up a private radio station is easy using a Raspberry Pi. This process should take thirty to sixty minutes to complete. By the end of this hour, you should be able to broadcast within a radius of around fifty meters. However, this is also facilitated by the use of a good antenna. 
+<div style="text-align: center; padding-bottom: 5%;">
+<img style="padding: 0; margin:0;" src="/engineering-education/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/raspberry21.jpg" alt="Raspberry Pi diagram"><br>
+<span style="font-size: 11px;">Figure 1: Sample Rapberry Pi <a href="https://unsplash.com/photos/raLeFIxXgDY">Raspberry Pi</a>, 2021.<span>
+</div>
+
+Materials required for this tutorial include:
+1.	Raspberry Pi.
```

Comment:
You have to be more specific on which Raspberry Pi we are using. i.e., Raspberry pi 3, 4?